.travis.yml

```diff
@@ -13,6 +13,10 @@ language: go
 go:
   - 1.11.5
 
+# Needed for Ansible Vault
+python:
+  - 2.7
```

Comment:
Does Ansible not support Python 3? I think 2 is EOL this year, right?

Reply:
Good catch.  Moving to the Python 3 RPM instead.

Reply:
@nickrobison-usds I could use a fresh approval, thanks!